database/gormdb_assignment.go

```diff
@@ -163,7 +162,7 @@ func (db *GormDB) CreateCriterion(query *pb.GradingCriterion) error {
 
 // UpdateCriterion updates the given criterion
 func (db *GormDB) UpdateCriterion(query *pb.GradingCriterion) error {
-	return db.conn.Model(query).
+	return db.conn.
```

Comment:
Can we add a test for this to satisfy Codecov?

Reply:
Done.